pkg/tcpip/transport/tcp/tcp_test.go

```diff
@@ -6561,3 +6561,71 @@ func TestKeepaliveWithUserTimeout(t *testing.T) {
 		t.Errorf("got c.Stack().Stats().TCP.EstablishedTimedout = %v, want = %v", got, want)
 	}
 }
+
+func TestIncreaseWindowOnReceive(t *testing.T) {
+	// This test ensures that the endpoint sends an ack,
+	// after recv() when the window grows to more than 1 MSS.
+	c := context.New(t, defaultMTU)
+	defer c.Cleanup()
+
+	// Set the window size such that a window scale of 4 will be
+	// used. Testing window scaling is outside scope of this test.
+	const wnd = 65535 * 10
```

Comment:
rename wnd w/ rcvBuf? Since we are setting the receiver buffer with this not the window itself.

Reply:
done